spec/spec.md

```diff
@@ -1753,6 +1763,15 @@ The analysis of protection against an attack can be further decomposed into thre
 
 Protection consists of either prevention or mitigation of both of the harm cases. The primary protection mechanisms for the controller include best practice key management techniques for maintaining root control authority, redundant confirmation of events by supporting components, and duplicity detection on the behavior of designated supporting components. The primary protection mechanism for the validator is duplicity detection on the behavior of supporting components.
 
+#### Key Event Receipt Log
+
+A Key Event Receipt Log consists of signed key events, keeping track of establishment events. To begin with the inception event and any number of rotation events. We call that this establishment subsequence.  
+The Key Event Receipt Logs are built from receipts of events signed by the witnesses of those events (called *commitments*); these are also append-only but not hash-chained.
+
```

Comment:
This is a bit confusing.  I think the definition of KERL should start with KEL, but that also includes receipts from witnesses.